### PR TITLE
Add layouts/application.html and welcome login form

### PR DESCRIPTION
**Pull request by a6ftcruton**
Testing out a login form for Rails to kickoff this OAuth journey. It's going to be mystical!

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <title>Skills Inventory</title>
+    <%= stylesheet_link_tag    "application" %>
+    <%= javascript_include_tag "vendor/modernizr" %>
+    <%= csrf_meta_tags %>
+  </head>
+
+  <body>
+    <%= yield %>
+    <%= javascript_include_tag "application" %>
+    <script>$('.flash').delay(4000).fadeOut(800);</script>
```
Comment (stevekinney) [inline app/views/layouts/application.html.erb]: Can we move this out of the view?

File: app/views/welcome.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+</div>
+
+<div id="login-form">
+  <form {{action 'authenticate' on='submit'}}>
+    <label for="identification">Login</label>
+    {{input value=identification placeholder='Enter Login'}}
+    <label for="password">Password</label>
+    {{input value=password placeholder='Enter Password' type='password'}}
+    <button type="submit">Login</button>
+  </form>
```
Comment (stevekinney) [inline app/views/welcome.html.erb]: I think you're writing Handlebars in your ERB template.

File: npm-debug.log [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+6 error package.json npm can't find a package.json file in your current directory.
+7 error System Darwin 13.3.0
+8 error command "node" "/usr/local/bin/npm" "install"
+9 error cwd /Users/Jim/Turing/skill-inventory-rails
+10 error node -v v0.10.32
+11 error npm -v 1.4.28
+12 error path /Users/Jim/package.json
+13 error code ENOPACKAGEJSON
+14 error errno 34
+15 verbose exit [ 34, true ]
```
Comment (stevekinney) [inline npm-debug.log]: This belongs in your `.gitignore`.

File: app/views/welcome.html.erb (hunk 2 above)
Comment (a6ftcruton) [inline app/views/welcome.html.erb]: I just saw this and feel stupid. It was a hasty pre-lunch push. :facepunch:

Comment (stevekinney) [inline app/views/welcome.html.erb]: Code review, FTW!

**Pull request merged**